Documentation/ApiOverview/Hooks/Events/Extbase/Persistence/EntityAddedToPersistenceEvent.rst

```diff
@@ -6,8 +6,9 @@
 EntityAddedToPersistenceEvent
 =============================
 
-Event which is fired after an object/entity was sent to persistence layer to be added,
-but before updating the reference index and current session.
+Event which is fired after an object/entity was persisted on add.
```

Comment:
now I'm confused: is the entity actually persisted at this point?

Reply:
Yes, it's already persisted:

https://github.com/TYPO3/typo3/blob/2f940fd527acfc719fbfdbe8a9f789ec18ba009a/typo3/sysext/extbase/Classes/Persistence/Generic/Backend.php#L632

The event is dispatched afterwards:

https://github.com/TYPO3/typo3/blob/2f940fd527acfc719fbfdbe8a9f789ec18ba009a/typo3/sysext/extbase/Classes/Persistence/Generic/Backend.php#L636

Exactly this fact was what I tried to clarify.

Reply:
maybe we need to rewrite the sentence ` before updating the reference index and adding the object to the persistence session.` because it sounds as if this means it's not added yet.

Reply:
Uh, that's what it already says? Not sure how to make this even clearer.

Reply:
I'll try and you can shout at me if I broke the meaning.

Reply:
@mbrodala how about now?

Reply:
I shouted already. ;-)

Reply:
Now it may contain a slight duplication, however, now it's understandable even for me ;)

Reply:
Yup, if in doubt, repeat. ;-)